File: if else/equalnum.java
/*#Check if two numbers are equal*/

import java.util.Scanner;

class equalnum{
    public static void main(String args[]){
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter first number:");
        int num1=sc.nextInt();
        System.out.println("Enter second number:");
        int num2=sc.nextInt();

        if(num1==num2){
            System.out.println("Both numbers are equal");
        }
        else{
            System.out.println("Numbers are not equal");
        }
    }
}

File: C/if else/equalnum.c
/*Check if two numbers are equal*/

#include <stdio.h>

int main() {
    int num1, num2;

    printf("Enter num1: ");
    scanf("%d", &num1);
    
    printf("Enter num2: ");
    scanf("%d", &num2);

    if (num1 > num2) {
        printf("num2 is smaller");
    }
    else if (num2 > num1) {
        printf("num1 is smaller");
    }
    else{
        printf("Two of them is equal");
    }

    return 0;
}